src/explorer/trialApp/TrialAppClient.ts

```diff
@@ -54,7 +54,21 @@ export class TrialAppClient implements IAppSettingsClient, IFilesClient {
     }
 
     public get defaultHostUrl(): string {
-        return `https://${this.metadata.url}`;
+        return this.metadata.url;
+    }
+    public get gitUrl(): string {
```

Comment:
Don't make me nit comment all of these Alex 😭

Reply:
Ok so we always put spaces between getters? Because I did that on purpose (I hate it). But I thought there are places we don't put spaces between getters. 

Since it seems like we DO want spaces between getters I will stop doing that (thank goodness).

Reply:
Oh, really?  Okay, I mean that's my preference too, so since I'm the one reviewing, let's put spaces 😂